src/getMachineId.py

```diff
@@ -0,0 +1,40 @@
+import boto3
+from decEncoder import *
+from boto3.dynamodb.conditions import Key, Attr
+
+"""Dynamo resource"""
+dynamodb = boto3.resource('dynamodb', region_name = 'us-east-2')
+machineTable = dynamodb.Table('Machines')
+
+"""Lambda handler function for /buildings/{buildingId}/machines/{machineId} API call
+   Returns machine with the machineId specified in the path
+   or 'Machine not found' error if machineId not found by get_item search
+"""
+def getMachineId(event, context):
+    """If machineId specified, assign it to variable,
+       use get_item to find it in machine table
+       put it in JSON format and return
+    """
+    machineId = None
+    if event.get('pathParameters'):
+        machineId = event.get('pathParameters').get('machineId')
+        response = machineTable.get_item(Key={'machineId': machineId})
```

Comment:
With just this, someone can put in a url like `/buildings/fakeBuilding/machines/1800-6-ave-blitman-hall-Dryer-08` and it will work. It's easier to keep it like this, but it produces unexpected behavior. 

Unfortunately, it doesn't look like `get_item` supports filters. I suggest you check the building ID of the item from `get_item`. If it matches what the user gave, then return, otherwise 404.